### PR TITLE
Added current protocol, data upload and data download feature 

### PR DESCRIPTION
**Pull request by renishnair-viper**
When we connect to any server we currently show the Visible location and Current IP along with that we added the below feature in UI.

1. Showing the currently selected protocol
2. Live usage of data download and data upload so users will be able to know the usage.

### CODE REVIEW

**Review by vimalraj-adikesavan — APPROVED** [review]

File: src/App/ViewModels/HomeViewModel.cs
Comment (latinvpn) [inline src/App/ViewModels/HomeViewModel.cs:1]: ME DA ERROR AL IMPLEMENTAR LA LINEA 77 SDK.DataTransferUpdate

File: src/App/Properties/Strings.ru.resx [hunk 1]
```diff
@@ -590,4 +590,13 @@
   <data name="Delete" xml:space="preserve">
     <value>удалить</value>
   </data>
+  <data name="CurrentProtocol" xml:space="preserve">
+    <value>Текущий протокол</value>
+  </data>
+  <data name="DataDownloaded" xml:space="preserve">
+    <value>Данные загружены</value>
```
Comment (rolikoff) [inline src/App/Properties/Strings.ru.resx]: Скачано

File: src/App/Properties/Strings.ru.resx [hunk 2]
```diff
@@ -590,4 +590,13 @@
...
+  </data>
+  <data name="DataUploaded" xml:space="preserve">
+    <value>Данные загружены</value>
```
Comment (rolikoff) [inline src/App/Properties/Strings.ru.resx]: Загружено

Comment (renishnair-viper) [reply src/App/Properties/Strings.ru.resx]: Resolved

File: src/App/Properties/Strings.ru.resx (hunk 1 above)
Comment (renishnair-viper) [reply src/App/Properties/Strings.ru.resx]: Resolved

**Review by rastapopoulos1 — APPROVED** [review]

**Review by rastapopoulos1 — APPROVED** [review]

**Review by rastapopoulos1 — APPROVED** [review]

**Review by vimalraj-adikesavan — APPROVED** [review]

**Pull request merged**